src/nvim/mbyte.c

```diff
@@ -353,12 +356,15 @@ int enc_canon_props(const char_u *name)
   int i;
 
   i = enc_canon_search(name);
-  if (i >= 0)
+  if (i >= 0) {
     return enc_canon_table[i].prop;
-  if (STRNCMP(name, "2byte-", 6) == 0)
+  }
```

Comment:
I'd add an else here manually to get rid of extra line, and same three lines below.